packages/web-components/@sfx/autocomplete/stories/index.ts

```diff
@@ -49,13 +70,19 @@ storiesOf('Components|Autocomplete', module)
     'Default',
     () => `
       <sfx-autocomplete results="${text('Autocomplete Results', JSON.stringify(results))}"></sfx-autocomplete>
+      <button @click=${this.dispatchAutocompleteResults}>
```

Comment:
I don't think any of these callbacks work anymore. We probably shouldn't be using `this` in this context.

Reply:
True, these don't work anymore and I noted that in the description. SFX-300 will resolve these issues as that incorporates the custom event handler addon made by Julia.